### PR TITLE
Make it possible to show blocks in the portlet column

### PR DESCRIPTION
**Pull request by maethu**
Implementation of a simple portlet which renders a simpleyout container.

### CODE REVIEW

File: ftw/simplelayout/browser/provider.py [hunk 1]
```diff
@@ -26,13 +26,21 @@ class BaseSimplelayoutExpression(object):
     fallbackview = ViewPageTemplateFile('templates/render_block_error.pt')
     structure = ViewPageTemplateFile('templates/structure.pt')
 
+    @property
+    def one_layout_one_column(self):
+        return [
+            {"cols": [{"blocks": []}
+                      ]
```
Comment (jone) [inline ftw/simplelayout/browser/provider.py]: I'd either add a trailing comma or move the closing bracket to the line above. Sorry, a little bit picky but it just popped into my eyes :wink:

File: ftw/simplelayout/portlets/portlet.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        for layout in portlet_container:
+            for columns in layout.values():
+                for column in columns:
+                    if len(column['blocks']):
+                        return True
+                    else:
+                        continue
+        return False
+
+    def update_simplelayout_settings(self, settings):
```
Comment (jone) [inline ftw/simplelayout/portlets/portlet.py]: :scissors: :question:

File: ftw/simplelayout/profiles/default/portlets.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<portlets>
+    <assignment
+      manager="plone.rightcolumn"
+      category="content_type"
+      key="ftw.simplelayout.ContentPage"
```
Comment (jone) [inline ftw/simplelayout/profiles/default/portlets.xml]: If I'd like to not have a portlet slot, how do I disable the portlet?

We could use a context assignment instead, what are the pros / cons?

File: ftw/simplelayout/tests/test_simplelayout_portlet.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+class TestSimplelayoutView(SimplelayoutTestCase):
+
+    layer = FTW_SIMPLELAYOUT_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        self.setup_sample_ftis(self.portal)
+        self.setup_block_views()
+
+        self.container = create(Builder('sample container'))
```
Comment (jone) [inline ftw/simplelayout/tests/test_simplelayout_portlet.py]: why are there sample types btw? couldn't we just use the normal page and text block?

**Comment by jone** [comment]
:+1: please rebase

File: ftw/simplelayout/profiles/default/portlets.xml (hunk 3 above)
Comment (maethu) [inline ftw/simplelayout/profiles/default/portlets.xml]: Disabling is possible by:
- Plone Control Panel -> Content types -> content specific portlets
- Blocking the portlet on a specific context


Context assignment is also possible - no it's enabled generally. Don't really know what is better. Probably not enabling it and a readme entry with a example portlet.xml is the best way.

File: ftw/simplelayout/tests/test_simplelayout_portlet.py (hunk 4 above)
Comment (maethu) [inline ftw/simplelayout/tests/test_simplelayout_portlet.py]: Sample types are in defined in ``SimplelayoutTestCase``. 

For Plone5 compatibility (mainly widget issues) We decided to use abstract Simplelayout types to test simplelayout features.

**Comment by maethu** [comment]
@jone updated

File: ftw/simplelayout/tests/test_simplelayout_portlet.py (hunk 4 above)
Comment (jone) [inline ftw/simplelayout/tests/test_simplelayout_portlet.py]: :+1:

**Pull request merged**